src/pip/_internal/commands/debug.py

```diff
@@ -85,7 +81,6 @@ def get_module_from_module_name(module_name):
 
 def get_vendor_version_from_module(module_name):
     # type: (str) -> str
```

Comment:
There seems to be a possibility of returning `None`, so shouldn't the return type be `Optional[str]`?

Reply:
So the return type of [getattr](https://github.com/python/typeshed/blob/master/stdlib/2and3/builtins.pyi#L1193) is `Any`, and I think because of that, both `str` and `Optional[str]` works and is accepted by mypy.

 I myself am not sure which one is more appropriate, perhaps `Optional[str]` is better because we can get `version = None` in `version = getattr(package, 'version', None)` . I will wait for others to chime in with their suggestions

Reply:
This should definitely be `Optional[str]`. The call site does actively consider the possibility of version being None as well.

Reply:
That makes sense. Fixed in https://github.com/pypa/pip/pull/8051/commits/78c0a7192aae3cc61060ca91e4276626bdc6cd86